src/frontends/onnx/frontend/src/op/reduce.cpp

```diff
@@ -94,6 +95,27 @@ const std::set<element::Type> supported_types_v1 =
     {element::u32, element::u64, element::i32, element::i64, element::f16, element::f32, element::f64};
 const std::set<element::Type> supported_types_v2 =
     {element::u32, element::u64, element::i32, element::i64, element::f16, element::f32, element::f64, element::bf16};
+const std::set<element::Type> supported_types_v3 = {element::u32,
```

Comment:
Hi,
I'm working on new opsets for ReduceLogSumExp, and would like to take this PR as example. 
I'm wondering whether inventing new versioning scheme for types is a good idea ? Wouldn't it be better to just put the constant supported_types into a namespace ?

Reply:
@mwilczy you mean why I didn't align it with an opset version? For example, supported_types_v1, v11, v13, etc. Or op::set_1::supported_types}, op::set_11::supported_types, etc. Did I get correct? I thought about it at first step but...

I motivated to do it as I did because I compared all differences as a table, and I found by text comparison between all versions of all Reduce* operations. A summary:

Types versioning isn't aligned between Reduce* versions. For example, ReduceMax/Min-13 has different list of supported types (added int8 and uint8) than all other Reduce*-13 operations. as I can see - opset-13 simultaneously introduced two different lists of supported types. I really don't like constructions like op::set_13::supported_types::v1 ;)

So, supported_types for ReduceMax/Min will look different. And in such case I may need to introduce more intermediate entities like: define four different lists in a common part and use it thru... "using support_types = supported_types_v3" for each Reduce*? I think it will overcomplicate the source without any benefits for reading. In my opinion it could be reasonable in case we would separate Reduce* operations for a set of files (maybe it will be done when we will have all implementations ready).

Hope, I've answered on your question...